Cell.js

```diff
@@ -0,0 +1,19 @@
+class Cell {
+    constructor(
+        row,
+        column,
+        isMine,
+        numMinesAround,
+        isRevealed,
+        flaggingOn
+    ) {
+        this.column = column;
+        this.row = row;
+        this.isMine = isMine;
+        this.numMinesAround = numMinesAround;
+        this.isRevealed = isRevealed;
+        this.flaggingOn = flaggingOn
```

Comment:
prefix boolean values with `is` (use the handy refactor replace tool)
```suggestion
        this.isFlaggingOn = isFlaggingOn
```

Reply:
done